contracts/factory/ITREXGateway.sol

```diff
@@ -102,6 +102,12 @@ interface ITREXGateway {
     /// event emitted whenever a TREX token has been deployed by the TREX factory through the use of the Gateway
     event GatewaySuiteDeploymentProcessed(address indexed requester, address intendedOwner, uint256 feeApplied);
 
+    /// event emitted whenever the gateway is paused by an admin
+    event GatewayPaused(address _userAddress);
```

Comment:
I didn't use the names "Paused" and "Unpaused" because they are used by the token contract. Otherwise, sharing the same event names as token events would cause ambiguity and collision on the indexer side.